openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -32,6 +38,111 @@ def get_identity(self, data):
             return None
 
 
+class UserProfileReadOnlySerializer(serializers.Serializer):
```

Comment:
The naming is tricky. Nimisha expressed that she doesn't feel like this is an account, even though that's what we called the endpoint. She argued that this is a high level representation of a user, so she would have the endpoint be ``/api/users/v1/users/andya``. Given this thought, maybe this could be simply ``UserReadOnlySerializer``. It is producing a read-only serialization of the user, regardless of where the information is coming from.